packages/admin/admin/src/actionLog/diffHeader/DiffHeader.sc.ts

```diff
@@ -0,0 +1,37 @@
+import { css } from "@emotion/react";
+import { Box, Typography } from "@mui/material";
+
+import { createComponentSlot } from "../../helpers/createComponentSlot";
+import { type DiffHeaderClassKey } from "./DiffHeader";
+
+export const Root = createComponentSlot(Box)<DiffHeaderClassKey>({
+    componentName: "DiffHeader",
+    slotName: "root",
+})(
+    ({ theme }) => css`
+        display: flex;
+        flex-direction: column;
+        align-items: start;
+        background-color: ${theme.palette.grey.A200};
+        padding: ${theme.spacing(2)};
+    `,
+);
+
+export const Version = createComponentSlot(Typography)<DiffHeaderClassKey>({
+    componentName: "DiffHeader",
+    slotName: "version",
+})(() => css``);
```

Comment:
Just FYI (doesn't actually matter): the styles-function can be omitted when no styles are needed 🙂 

```suggestion
})();
```